pandas/core/groupby.py

```diff
@@ -929,6 +929,10 @@ def mean(self):
         Compute mean of groups, excluding missing values
 
         For multiple groupings, the result index will be a MultiIndex
+
+        Do not call this function via the numpy library. numpy's mean
```

Comment:
Call it an overreaction if you want, but this PR wouldn't have happened if it weren't for the fact that people are trying to call `pandas` functions via the `numpy` library, and something tells me people will continue to try doing so.  Do you have an alternative suggestion besides documentation for making it clear to users that this particular function shouldn't be called via `numpy`?